app/__init__.py

```diff
@@ -40,6 +40,13 @@ def create_app():
         api.add_resource(
             routes.Attachments,
             '/attachments',
+        )
+
+        api.add_resource(
+            routes.Attachment,
+            '/attachments',
+            '/attachments/<string:hash_id>',
+            '/attachments/<string:hash_id>/attachment',
             '/attachments/<string:hash_id>/attachment/<string:attachment_id>',
         )
```

Comment:
This still does not feel right. We should have one endpoint not four? 
I see this as four different endpoints

Already has an endpoint for: '/attachments'

Why all three? 
'/attachments/<string:hash_id>', // get all attachment for a user?
'/attachments/<string:hash_id>/attachment', // does not get anything?
'/attachments/<string:hash_id>/attachment/<string:attachment_id>', // get single attachment for a user? 

```
api.add_resource(
    routes.Attachments,
    '<string:hash_id>/attachments',
)
api.add_resource(
    routes.Attachment,
    '<string:hash_id>/attachments/<string:attachment_id>', or use query strings '/attachments/<string:attachment_id>?hash_id=<string:hash_id>'
)
```

Reply:
So my thought here is that everything relates to a user, as far as i know. With other words you cannot get or post attachments without a user connected to them? If this is the case then the {user_hash}/attachments/{attachment_id} feels more natural :)

Reply:
Tar ett snakk

Reply:
Ska vi ta tag i detta imorgon kanske?

Reply:
Anledningen till detta är att slippa få fula html-fel som svar om du går mot en endpoint som inte finns.
Ja, det finns säkert bättre sätt att lösa detta på som jag inte vet just nu.

Snakkes!